easybuild/easyconfigs/h/hector/hector-2.5.0-GCC-11.2.0.eb

```diff
@@ -0,0 +1,40 @@
+# Contribution from Imperial College London, UK
+# uploaded by J. Sassmannshausen
+
+easyblock = 'MakeCp'
+
+name = 'hector'
+version = '2.5.0'
+
+homepage = 'https://github.com/JGCRI/hector'
+description = """This is the repository for Hector, an open source, object-oriented, 
+simple global climate carbon-cycle model. It runs essentially instantaneously while 
+still representing the most critical global scale earth system processes, and is 
+one of a class of models heavily used for for emulating complex climate models 
+and uncertainty analyses."""
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+toolchainopts = {'openmp': True}
+
+source_urls = ['https://github.com/JGCRI/hector/archive']
+sources = [{'download_filename': 'rcmip-tier1.tar.gz', 'filename': 'v%(version)s.tar.gz'}]
+checksums = ['457fec10f0c59e65b1b00c5d6c9aa66b38cd94c0c5a89444d059bebeb3fba3c9']
+
+dependencies = [
+    ('Boost', '1.79.0'),
+]
+
+buildopts = 'hector'
+
+files_to_copy = [
+    (['src/hector'], 'bin'),
+    (['src/libhector.a'], 'lib'),
+    'R', 'man', 'LICENSE.md', 'README.md', 'DESCRIPTION', 'scripts', 'data-raw', 'inst',
+]
+
+sanity_check_paths = {
+    'files': ['bin/hector'],
+    'dirs': [],
+}
+
+moduleclass = 'tools'
```

Comment:
```suggestion
moduleclass = 'geo'
```
?

Reply:
Looks better to me. I was not sure what to choose so I went for `tools`. I have updated that to `geo`. Thanks!